src/inverse-client.ts

```diff
@@ -439,23 +320,19 @@ export class RestClient {
   getLastFundingRate(params: {
     symbol: string;
   }): GenericAPIResponse {
-    return this.requestWrapper.get('open-api/funding/prev-funding-rate', params);
+    return this.requestWrapper.get('v2/private/funding/prev-funding-rate', params);
```

Comment:
Seems this one should be `v2/public/funding/prev-funding-rate`:
https://bybit-exchange.github.io/docs/inverse/#t-fundingrate